CHANGELOG.md

```
@@ -1,5 +1,24 @@
 # Changelog
 
+## EasyCord v5.44.3 - 2026-06-14
+
+### Fixed
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)